packages/nc-gui/utils/validation.ts

```diff
@@ -98,13 +98,15 @@ export const fieldRequiredValidator = () => {
   }
 }
 
-export const fieldLengthValidator = (sqlClientType: string) => {
+export const fieldLengthValidator = (_sqlClientType: string) => {
   return {
     validator: (rule: any, value: any) => {
       const { t } = getI18n().global
 
-      // no limit for sqlite but set as 255
-      let fieldLengthLimit = 255
+      const fieldLengthLimit = 255
+
+      /*
+      Now we allow 255 for all databases, truncate will be handled by backend for column_name
```

Comment:
i would suggest to put this comment above L106. then delete the below block. or you may include those numbers in the comment for reference.

```
if (sqlClientType === 'mysql2' || sqlClientType === 'mysql') {
  fieldLengthLimit = 64
} else if (sqlClientType === 'pg') {
  fieldLengthLimit = 59
} else if (sqlClientType === 'mssql') {
  fieldLengthLimit = 128
}
```